website/blog/2024-02-27-release-package-updates/index.mdx

```diff
@@ -0,0 +1,98 @@
+---
+slug: package-updates
+title: Glasskube v0.0.3 — Introducing Package Updates
+description: "Version 0.0.3, launched on February 27th: 10 contributors, 64 commits, introducing package update management."
+authors: [ pmig ]
+tags: [ glasskube, release, kubernetes ]
+---
+
+import Install from '../../src/partials/_install.mdx';
+
+*Glasskube v0.0.3 was released on February 27th, introducing package updates and many useful features for improved CLI and GUI experience.*
+
+
+Glasskube is fully open-source. Support us by leaving a star: [⭐ `glasskube/glasskube` ⭐](https://github.com/glasskube/glasskube/)
+
+## 🙏 Special thanks to all our contributors 🥰
+
+Once again, we can't thank our community enough for their valuable input and exciting contributions to Glasskube.
+We are happy you choose to be part of our journey in making Kubernetes package management easier for everyone.
+
+**This release includes:**
+
+- 👥 a total of 10 contributors
+- 🛠️ 64 commits
+- 💥 no breaking changes
+
+Watch our release video to get an overview of what has changed:
+
+<iframe width="100%" height="460" src="https://www.youtube-nocookie.com/embed/fbi2fdQiRrs?si=cSBxS_0R2VyrkbYL"
+        title="YouTube video player" frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Most notable changes
+
+### Package Updates
+
+Updating packages is one of the most essential features of any package manager. That's why most of our efforts in the past two weeks
+have gone into the support for such updates via CLI and GUI. It's as easy as typing `glasskube update` into the command line,
+or a button click in the GUI.
+
+During package installation you now can decide whether you want automatic updates for the package, in which case the
+Glasskube package operator will take care of keeping you up to date at all times. You can of course opt out of this feature
+if you prefer to handle updates manually.
+
+![Package Updates](./package-updates.png)
+
+For technical details, please have a look at the [package operator documentation](/docs/components/package-operator/#handling-package-updates).
```

Comment:
I would love to also see a short section about installing a specific version with the new `--version` flag